e2e/testinfra/terraform/common/registries.tf

```diff
@@ -32,3 +32,13 @@ resource "google_artifact_registry_repository" "gar" {
   description   = "A private AR repository used for Config Sync e2e tests"
   format        = "DOCKER"
 }
+
+resource "google_secure_source_manager_instance" "ssm-instance" {
+    location = "us-central1"
+    instance_id = "ssm-instance"
```

Comment:
Let's give this a name indicative of it's intended purpose, e.g. `configsync-test` or something similar